Tasks/Kaplunou/Task2/Task2.py

```diff
@@ -0,0 +1,83 @@
+import time
+Choice=input("do you want input time or use current time? Yes or No: ")
+if Choice == "Yes":
+    Time = input("Input time in format hh:mm: ")
+    Time=Time.split(":")
+    CurentHour = int(Time[0])
+    CurentMinute = int(Time[1])
+else:
+    seconds = time.time()
+    CurrentTime = time.localtime(seconds)
+    CurentHour = int(CurrentTime.tm_hour)
+    CurentMinute = int(CurrentTime.tm_min)
+print(CurentHour,CurentMinute)
+digits=[' ', 'one', 'two', 'three', 'four', 'five', 'six', 'seven', 'eight', 'nine']
+teens = ['ten', 'eleven', 'twelve', 'thirteen', 'fourteen', 'fifteen', 'sixteen', 'seventeen', 'eighteen', 'nineteen']
+tens = [' twenty ', ' thirty ', ' forty ', ' fifty ']
```

Comment:
oh noes, it should be russion as explicitly stated in the task description, -20 points for a missing part of the task